api/v1alpha1/perconaservermysql_types.go

```diff
@@ -275,8 +275,9 @@ type ServiceExposeTogglable struct {
 type StatefulAppState string
 
 const (
-	StateInitializing StatefulAppState = "initializing"
-	StateReady        StatefulAppState = "ready"
+	StateInitializing     StatefulAppState = "initializing"
+	StateReady            StatefulAppState = "ready"
+	StateReplicationError StatefulAppState = "replication error"
```

Comment:
Can we have just a replication error here, what about other errors if any? Should we have a more general error in here if there are more things that can break besides replication?

Reply:
@nonemax just "error" might be better, since we have the same for PSMDB and PXC operators